### PR TITLE
[enterprise-3.7]Added async release notes for RHBA-2018:2337 + latest-tag bump

### PR DESCRIPTION
**Pull request by ahardin-rh**
To be released 2018-08-07

cc @vikram-redhat @tmorriso-rh

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@ahardin-rh make sure to backport the release note changes to the master branch.

File: release_notes/ocp_3_7_release_notes.adoc [hunk 1]
```diff
@@ -4040,3 +4040,91 @@ To upgrade an existing {product-title} 3.6 or 3.7 cluster to this latest
...
+
+[[ocp-3-7-61-bug-fixes]]
+==== Bug Fixes
+
+* Due to corrupt file chunk buffers, Fluentd was blocked processing messages
+ until  the buffer was removed. This bug fix introduces a handler to remove
+ corrupt  messages from the processing work flow. Corrupt messages are now sent
+ to a dead  letter queue while continuing to process other messages and the
+ pipleline is no  longer blocked.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1511116[*BZ#1511116*])
```
Comment (tmorriso-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: The spacing looks odd in the comments view, but not in the whole file view. For example, "until  the buffer" and "a dead  letter" and "no  longer blocked". You may want to see if it looks the same in your view.

File: release_notes/ocp_3_7_release_notes.adoc [hunk 2]
```diff
@@ -4040,3 +4040,91 @@ To upgrade an existing {product-title} 3.6 or 3.7 cluster to this latest
...
+
+* Node labeling for Fluentd was done via a script run from the *_/tmp_* directory.
+If *_/tmp_* was mounted with the `noexec` option, then Ansible could not execute
+the script and fails to complete the playbook. The execution is now moved out of
+a script and into a `shell` Ansible task. The playbook run is now completed
+successfully.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1600685[*BZ#1600685*])
+
+* There was an undefined variable in the Fluentd role and the deployment failed
+trying to label the nodes. With this bug fix,  the variable is replaced with the
```
Comment (tmorriso-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: s/failed trying/failed when trying

File: release_notes/ocp_3_7_release_notes.adoc [hunk 3]
```diff
@@ -4040,3 +4040,91 @@ To upgrade an existing {product-title} 3.6 or 3.7 cluster to this latest
...
+correct one to the `oc` client binary. Logging deployment is  now possible and
+properly labels the nodes.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1601307[*BZ#1601307*])
+
+* Recently, `cloudResourceSyncManager` was implemented ,which continuously fetched
```
Comment (tmorriso-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: s/implemented ,which/implemented, which

File: release_notes/ocp_3_7_release_notes.adoc [hunk 4]
```diff
@@ -4040,3 +4040,91 @@ To upgrade an existing {product-title} 3.6 or 3.7 cluster to this latest
...
+node addresses from cloud providers. The kubelet then received node addresses
+from the `cloudResourceSyncManager`. At the time of node registration or kubelet
+start, the kubelet fetches node addresses in a blocking loop from
+`cloudResourceSyncManager`. The issue was that `cloudResourceSyncManager` was
+not started before the kubelet had started fetching node addresses from it for
+the first time. Due to this, the kubelet got stuck in the blocking loop and
+never returned. It caused node failures at network level, and no node could be
```
Comment (tmorriso-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: s/at network level/at the network level

File: release_notes/ocp_3_7_release_notes.adoc [hunk 5]
```diff
@@ -4040,3 +4040,91 @@ To upgrade an existing {product-title} 3.6 or 3.7 cluster to this latest
...
+`cloudResourceSyncManager` is now started early in the kubelet startup process so
+that the kubelet does not get blocked on it and `cloudResourceSyncManager` is
+always started.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1601378[*BZ#1601378*])
+
+[[ocp-3-7-61-enhancements]]
+==== Enhancements
+
+* This feature allows the number of indices and replicas to be configured using
+environment variables. The logs collected for infra services consumes a large
```
Comment (tmorriso-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: s/consumes/consume

File: release_notes/ocp_3_7_release_notes.adoc [hunk 6]
```diff
@@ -4040,3 +4040,91 @@ To upgrade an existing {product-title} 3.6 or 3.7 cluster to this latest
...
+portion of the available disk space. Spreading the data across available nodes
+by modifying the replica and shard settings allow Elasticsearch to better
+support these large amounts of data. This feature results in improved
+performance in Elasticsearch when there are large amounts of data from infra
+Services.
```
Comment (tmorriso-rh) [inline release_notes/ocp_3_7_release_notes.adoc]: Check the spelling for infra services - services vs. Services

**Comment by tmorriso-rh** [comment]
@ahardin-rh A few comments.

**Comment by ahardin-rh** [comment]
@tmorriso-rh Thanks! All fixed. :fireworks:

**Pull request merged**